### ISSUE TITLE
Text recognition: Split wide crops parameter highly decrease performances

### ISSUE THREAD

**Issue by jonathanMindee** [mindee/doctr#592]
### 🚀 The feature

Fine tune the wide crops splitting method

### Motivation, pitch

Hey!

It's not a bug per se, because my code is running well. But I retrained a text recognition model, and the performances on my test set were very low compared to my validation set (both from the same dataset).

I digged a bit into the code, and noticed thes lines:
```
remapped = False
if self.split_wide_crops:
    new_crops, crop_map, remapped = split_crops(crops, self.critical_ar, self.target_ar, self.dil_factor)
    if remapped:
        crops = new_crops
```

I commented them, and the performances were back to what's expected.

I understand the motivation on this feature, but i noticed that it creates a lot of characters repetition in the output strings.

Example:
"AZZ812ZZF21" -> "AZZ81212ZZF21"

It's behaving as the cut was adding some padding, and some characters are recognized twice, leading to a repetition when reconstructing the final output.

I can send you a few sample data on Mindee's slack community if you need it.

**Comment by charlesmindee** [comment]
Hi @jonathanMindee, it would be great to have samples indeed on the slack community!
Thanks for that!

**Comment by jonathanMindee** [comment]
Hi @charlesmindee: Done on the docTR channel

**Comment by fg-mindee** [comment]
Coming back to this, should we then change the hyperparams of split/merge of crops to close this issue?

**Comment by charlesmindee** [comment]
As it turns out, this doesn't occur on our pretrained crnn. 
I tried a target_ar of 6, it is slightly better so I will open a PR to close this issue.
